### PR TITLE
Made a landing page

### PR DESCRIPTION
**Pull request by cuiran**

### CODE REVIEW

**Comment by Bachmann1234** [comment]
@cuiran can we get a screenshot?

File: diy_asylum_frontend/src/App.js [hunk 1]
```diff
@@ -23,10 +24,15 @@ class App extends Component {
       <IntlProvider locale={locale} key={locale} messages={messages}>
         <div className="App">
           <Router>
-            <Route
-              path="/:filter?"
-              render={({ match }) => <MainView filter={match.params.filter} />}
-            />
+            <switch>
```
Comment (JeuelyFish) [inline diy_asylum_frontend/src/App.js]: Was this working for you on localhost? 
I needed to switch these to `<Switch>...</Switch>` and then import update line #2 to also import Switch

File: diy_asylum_frontend/src/App.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React, { Component } from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
```
Comment (JeuelyFish) [inline diy_asylum_frontend/src/App.js]: `import { BrowserRouter as Router, Route, Switch } from "react-router-dom";`

(related to comment below)

File: diy_asylum_frontend/src/App.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React, { Component } from "react";
...
 import { connect } from "react-redux";
 import MainView from "./components/MainView";
+import LandingPage from "./components/LandingPage";
```
Comment (JeuelyFish) [inline diy_asylum_frontend/src/App.js]: import { BrowserRouter as Router, Route, Switch } from "react-router-dom";

Related to comment below in same file

File: diy_asylum_frontend/src/App.js (hunk 1 above)
Comment (cuiran) [reply diy_asylum_frontend/src/App.js]: It was working for me on localhost without the import.

File: diy_asylum_frontend/src/components/LandingPage/landing.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  body {
+    padding-top: 56px;
+  }
+}
+
+.business-header {
+  height: 50vh;
+  min-height: 300px;
+  background: url("../../img/landing_skygrass.png") center no-repeat scroll;
+  -webkit-background-size: auto;
```
Comment (JeuelyFish) [inline diy_asylum_frontend/src/components/LandingPage/landing.css]: fun fact!
The create-react-app already comes with post-processing CSS prefixes. https://facebook.github.io/create-react-app/docs/post-processing-css

In other words, you don't have to write all of the "webkit" "moz" or "o" prefixes to background-size.
A single `background-size: auto;` will do the trick

File: diy_asylum_frontend/src/components/LandingPage/index.js [hunk 5]
```diff
@@ -0,0 +1,166 @@
+import React, { Component } from "react";
+import "./landing.css";
+
+const LandingPage = () => (
+  <div>
+    <body>
```
Comment (JeuelyFish) [inline diy_asylum_frontend/src/components/LandingPage/index.js]: This `<body`> tag is throwing an error for me in the console.
To see the devtool console, just open this page in chrome and press `Command+Option+I` and make sure the `console` tab is selected.

You should see an error at the top that says ```
validateDOMNesting(...): <body> cannot appear as a child of <div>```

There are also some other warnings and errors in there that would be great if you could comb thru and sort out!

**Pull request merged**